waiter/src/waiter/scheduler/marathon.clj

```diff
@@ -456,8 +457,11 @@
      :syncer (retrieve-syncer-state-fn)})
 
   (validate-service [_ service-id]
-    (let [{:strs [run-as-user]} (service-id->service-description-fn service-id)]
-      (authz/check-user authorizer run-as-user service-id))))
+    (let [{:strs [ports run-as-user] :as service-description} (service-id->service-description-fn service-id)]
+      (authz/check-user authorizer run-as-user service-id)
+      (when (and (scheduler/http2-service? service-description) (ports < 2))
```

Comment:
Marathon requires an HTTP/2 service to listen on `PORT1` for health checks with HTTP/1 requests.